__tests__/Unit/Components/availability-panel/drag-drop-context/index.test.tsx

```diff
@@ -0,0 +1,91 @@
+import DragDropContextWrapper from '@/components/availability-panel/drag-drop-context';
+import { tasks, TASK } from '../../../../../__mocks__/db/tasks';
+import { renderWithProviders } from '../../../../../src/test-utils/renderWithProvider';
+
+export const urlPattern =
+    /(https:\/\/www\.|http:\/\/www\.|https:\/\/|http:\/\/)?[a-zA-Z]{2,}(\.[a-zA-Z]{2,})(\.[a-zA-Z]{2,})?\/[a-zA-Z0-9]{2,}|((https:\/\/www\.|http:\/\/www\.|https:\/\/|http:\/\/)?[a-zA-Z]{2,}(\.[a-zA-Z]{2,})(\.[a-zA-Z]{2,})?)|(https:\/\/www\.|http:\/\/www\.|https:\/\/|http:\/\/)?[a-zA-Z0-9]{2,}\.[a-zA-Z0-9]{2,}\.[a-zA-Z0-9]{2,}(\.[a-zA-Z0-9]{2,})?/g;
```

Comment:
What pattern are we matching here?

Reply:
> What pattern are we matching here?
i changed that pattern with a simple relative url pattern please check it